CONTRIBUTING.md

```diff
@@ -16,82 +16,81 @@
       - [Run the tests manually](#run-the-tests-manually)
       - [Edit the tests](#edit-the-tests)
       - [Inspecting test results](#inspecting-test-results)
-  - [Yalcing into platform-internal](#yalcing-into-platform-internal)
+  - [Yalc into platform-internal](#yalcing-into-platform-internal)
   - [Releasing Elements](#releasing-elements)
   - [Versioning Guidelines](#versioning-guidelines)
   - [Merging into main](#merging-into-main)
```

Comment:
Need to make sure these all match the updated headings and links, otherwise, they won't work.